kernel/modops.ml

```diff
@@ -403,7 +403,11 @@ let inline_delta_resolver env inl mp mbid mtb delta =
 	    | Undef _ | OpaqueDef _ -> l
 	    | Def body ->
 	      let constr = Mod_subst.force_constr body in
-	      add_inline_delta_resolver kn (lev, Some constr) l
+              let ctx = match constant.const_universes with
```

Comment:
Just nitpicking, isn’t there a helper function doing that already?

Reply:
I don't remember one.

Reply:
Declareops.constant_polymorphic_context